src/js/containers/LimitOrder/LimitOrderModals/OrderDetailsModal.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import { connect } from "react-redux";
+import { getTranslate } from "react-localize-redux";
+import ReactTable from "react-table";
+
+import { Modal } from "../../../components/CommonElement";
+import { LimitOrderSubmit } from "../../LimitOrder";
+import {getFormattedDate} from "../../../utils/common";
+import * as converters from "../../../utils/converter";
+import {displayNumberWithDot} from "../../../utils/converter";
```

Comment:
This import is redundant, please use import * as converters from "../../../utils/converter";

Reply:
Updated, using converters only